plutus-playground-server/usecases/CrowdFunding.hs

```diff
@@ -67,7 +68,7 @@ data CampaignAction = Collect | Refund
 PlutusTx.makeIsData ''CampaignAction
 PlutusTx.makeLift ''CampaignAction
 
-type CrowdfundingSchema =
+type Schema =
```

Comment:
I feel like this needlessly deviates from the `plutus-use-cases` version. I think we should try and keep them as similar as possible.

Reply:
Agreed. Fixing...